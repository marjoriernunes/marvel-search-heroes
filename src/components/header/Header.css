.title {
    text-transform: uppercase;
    font-weight: 700;
    color: #404040;
    margin: 0;
    padding-top: 2rem;
}

.subtitle {
    color: #8c8c8c;
    margin: 0;
    padding-bottom: 2rem;
}