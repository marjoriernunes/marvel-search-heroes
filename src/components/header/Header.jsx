const Header = () => {
    return (
        <>
            <h1>explore o universo</h1>
            <p>Mergulhe no domínio deslumbrante de todos os personagens clássicos que você ama - e aqueles que você descobrirá em breve!</p>
        </>
    )
}

export default Header;