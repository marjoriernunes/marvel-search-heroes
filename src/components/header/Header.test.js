import {render} from '@testing-library/react'
import Header from './Header.jsx'

test('renders correctly', () =>{
    const utils = render(<Header/>)
    expect(utils.container).toMatchSnapshot()
});
