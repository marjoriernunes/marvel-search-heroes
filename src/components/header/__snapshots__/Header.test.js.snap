// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    explore o universo
  </h1>
  <p>
    Mergulhe no domínio deslumbrante de todos os personagens clássicos que você ama - e aqueles que você descobrirá em breve!
  </p>
</div>
`;
